easybuild/easyconfigs/a/alleleCount/alleleCount-4.0.0-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,45 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/easybuilders/easybuild
+# 
+# Author: Jonas Demeulemeester
+# The Francis Crick Institute, London, UK
+##
+
+easyblock = 'CmdCp'
+
+name = 'alleleCount'
+version = '4.0.0'
+
+homepage = 'http://cancerit.github.io/alleleCount/'
+description = """ The alleleCount package primarily exists to prevent code duplication
+between some other projects, specifically AscatNGS and Battenberg. As of v4 the perl 
+code wraps the C implementation of allele counting code for BAM/CRAM processing. """
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+sources = ['v%(version)s.tar.gz']
+source_urls = ['https://github.com/cancerit/%(name)s/archive/v%(version)s']
+checksums = ['3841dee52b79ec520d5a2faa83dff048badb46dafddb0053b7553eb20aa7fea1']
+
+cmds_map = [('.*', "./setup.sh . ${EBROOTPERL}/lib/perl5")]
+
+# HTSlib 1.7 is built, used and removed again in the provided setup script
+dependencies = [
+    ('Perl', '5.26.1')
```

Comment:
@jdemeul Looks like `cURL` is missing as a dependency, since I'm running into errors like:

```
configure: error: libcurl library not found
```